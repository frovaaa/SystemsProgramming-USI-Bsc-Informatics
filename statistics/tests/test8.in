+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
+1    +1.0    -1.1   1     2  100 100.000  200.0 200.0 200.0 200.0
