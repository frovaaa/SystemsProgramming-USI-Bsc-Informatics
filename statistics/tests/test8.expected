-1.100000 91.263741 200.000000
